package.json

```diff
@@ -131,17 +130,56 @@
       }
     }
   },
+  "eslintConfig": {
```

Comment:
do we need to add "globals" that were in .eslintrc file previously?

Reply:
I don't think so. I've added `plugin:jest/recommended`. It should replace the globals we had before.